cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/store/RepositorySourceControlMetadataStore.java

```diff
@@ -151,12 +150,24 @@ public int scan(ScanSourceControlMetadataRequest request,
    * @param lastSynced The timestamp of the last git pull/push of the entity
    * @throws IOException If an I/O error occurs while writing the source control metadata.
    */
-  public void write(ApplicationReference appRef, Boolean isSynced,
+  public void write(ApplicationReference appRef, boolean isSynced,
       Long lastSynced) throws IOException {
     StructuredTable repoTable = getRepositorySourceControlMetadataTable();
     repoTable.upsert(getAllFields(appRef, isSynced, lastSynced));
   }
 
+  /**
```

Comment:
I can't comment on line 120. But there is 21 lines of duplicated code detected by IntelliJ with the `NamespaceSourceControlMetadataStore` class. Why are we duplicating code?

Reply:
This two classes are handilng crud functions for two different (but similar) tables. Hence the duplication.